### PR TITLE
docs(auth, phone): add link to upstream android setup instructions

### PR DESCRIPTION
**Pull request by dqii**
### Checklist

- I read the [Contributor Guide](../CONTRIBUTING.md) and followed the process outlined there for submitting PRs.
  - [ x] Yes
- My change supports the following platforms;
  - [ x] `Android`
  - [ ] `iOS`
- My change includes tests;
  - [ ] `e2e` tests added or updated in `packages/\*\*/e2e`
  - [ ] `jest` tests added or updated in `packages/\*\*/__tests__`
- [ ] I have updated TypeScript types that are affected by my change.
- This is a breaking change;
  - [ ] Yes
  - [ x] No

### Test Plan

No tests, just adding some documentation
---

Think `react-native-firebase` is great? Please consider supporting the project with any of the below:

- 👉 Star this repo on GitHub ⭐️
- 👉 Follow [`React Native Firebase`](https://twitter.com/rnfirebase) and [`Invertase`](https://twitter.com/invertaseio) on Twitter

### CODE REVIEW

**Review by mikehardy — APPROVED** [review]
Nice! This is a good catch on a missing part, I think the SafetyNet stuff especially is semi-new upstream vs the docs and now we really need to refer upstream
Thanks for taking the time to post this, I'm sure it will help future people

File: docs/auth/phone-auth.md [hunk 1]
```diff
@@ -25,6 +25,11 @@ Phone auth requires app verification, and the automatic app verification process
 
 For reliable automated testing, you may want to disable both automatic and fallback reCAPTCHA app verification for your app. To do this, [you may disable app verification in AuthSettings](https://rnfirebase.io/reference/auth/authsettings#appVerificationDisabledForTesting) prior to calling any phone auth methods.
 
+
+# Android Setup
```
Comment (mikehardy) [inline docs/auth/phone-auth.md:29]:
```suggestion
# Android Setup
```

**Comment by mikehardy** [comment]
This will need the CLA signed before we can merge, even though it's tiny - you may follow the details link on the CLA action or the ones in the comment from the CLA bot to get it signed, thank you!

**Comment by dqii** [comment]
Just submitted I think! :) Sorry about that

**Comment by mikehardy** [comment]
boom 💥  😄  thanks

**Pull request merged**